cloudfoundry-example/README.md

```diff
@@ -0,0 +1,28 @@
+# Vert.x on Cloud Foundry
+
+This demo shows a very simple hello world Vert.x project for [Cloud Foundry](https://www.cloudfoundry.org/). It has a simple HTTP server which simply serves the /webroot/index.html.
+
+This demo uses [Vert.x 3.5.0](http://vertx.io) and is packed using the official [CloudFoundry java-buildpack](https://github.com/cloudfoundry/java-buildpack). The `manifest.yml` specifies the app's memory to be 768MB because any lower and the java-buildpack throws an error that it can't allocate enough heap space.
```

Comment:
> specifies the app's memory to be 768MB because any lower and the java-buildpack throws an error that it can't allocate enough heap space

What? Can you confirm this is not an error.

Reply:
There is a mention of memory constraints [here](https://docs.cloudfoundry.org/buildpacks/java/#memory-constraints) and [here](https://www.cloudfoundry.org/just-released-java-buildpack-4-0/). "One major result of accounting for all of these memory regions is the likelihood that applications which previously ran in small containers (512M or less) will no longer be able to."